metaflow/client/core.py

```diff
@@ -540,16 +548,17 @@ def _unpickle_284(self, data):
         )
 
     def _unpickle_2124(self, data):
```

Comment:
need to add another version of course. So we can unpickle all versions of objects.